### PR TITLE
Sync Coingecko Api from scaffold-eth-stark

### PR DESCRIPTION
**Pull request by Buuh2511**
# Sync Coingecko Api from scaffold-eth-stark

## Types of change

- [ ] Feature
- [ ] Bug
- [x] Enhancement

## Comments (optional)
- Coingecko api sync created by Iossocket in eth-stark repo

### CODE REVIEW

**Comment by iossocket** [comment]
Hi @Buuh2511 , would you mind to check this pull request https://github.com/Scaffold-Stark/scaffold-stark-2/pull/353, it changed the price api trigger and cache behaviour, also move the api call to server side

**Comment by Buuh2511** [comment]
yes sir @iossocket, i will check PR #353

File: packages/nextjs/contracts/deployedContracts.ts
Comment (Nadai2010) [inline packages/nextjs/contracts/deployedContracts.ts:1]: This file is auto-generated, we will not upload these changes

**Review by Nadai2010 — CHANGES_REQUESTED** [review]

File: packages/nextjs/contracts/deployedContracts.ts
Comment (Buuh2511) [reply packages/nextjs/contracts/deployedContracts.ts:1]: yes sir

Comment (Nadai2010) [reply packages/nextjs/contracts/deployedContracts.ts:1]: I think everything in this PR has been fixed in #353 by @iossocket , apart from the deployedContract.ts it should always be uploaded empty.

what is the difference vs PR353?

Comment (Buuh2511) [reply packages/nextjs/contracts/deployedContracts.ts:1]: u are right sir, i just do a sync from scaffold-eth-stark repo to scaffold-stark-2 for api coingecko feature. then i think we can move on PR #353 and close this PR

Comment (Nadai2010) [reply packages/nextjs/contracts/deployedContracts.ts:1]: Perfect sir, we see that the CORS errors are also resolved, let's wait for the 353 to be merged and close this PR, and sync with scaffold-eth-stark

Comment (Nadai2010) [reply packages/nextjs/contracts/deployedContracts.ts:1]: Hi @Buuh2511, I think we can close this PR

Comment (Buuh2511) [reply packages/nextjs/contracts/deployedContracts.ts:1]: yes sir, im going to close this PR

**Pull request closed without merging**